docs/extending/extending_draftail.md

```diff
@@ -329,17 +325,203 @@ To fully complete the demo, we can add a bit of JavaScript to the front-end in o
 
 ```javascript
 document.querySelectorAll('[data-stock]').forEach((elt) => {
-    const link = document.createElement('a');
-    link.href = `https://finance.yahoo.com/quote/${elt.dataset.stock}`;
-    link.innerHTML = `${elt.innerHTML}<svg width="50" height="20" stroke-width="2" stroke="blue" fill="rgba(0, 0, 255, .2)"><path d="M4 14.19 L 4 14.19 L 13.2 14.21 L 22.4 13.77 L 31.59 13.99 L 40.8 13.46 L 50 11.68 L 59.19 11.35 L 68.39 10.68 L 77.6 7.11 L 86.8 7.85 L 96 4" fill="none"></path><path d="M4 14.19 L 4 14.19 L 13.2 14.21 L 22.4 13.77 L 31.59 13.99 L 40.8 13.46 L 50 11.68 L 59.19 11.35 L 68.39 10.68 L 77.6 7.11 L 86.8 7.85 L 96 4 V 20 L 4 20 Z" stroke="none"></path></svg>`;
+  const link = document.createElement('a');
+  link.href = `https://finance.yahoo.com/quote/${elt.dataset.stock}`;
+  link.innerHTML = `${elt.innerHTML}<svg width="50" height="20" stroke-width="2" stroke="blue" fill="rgba(0, 0, 255, .2)"><path d="M4 14.19 L 4 14.19 L 13.2 14.21 L 22.4 13.77 L 31.59 13.99 L 40.8 13.46 L 50 11.68 L 59.19 11.35 L 68.39 10.68 L 77.6 7.11 L 86.8 7.85 L 96 4" fill="none"></path><path d="M4 14.19 L 4 14.19 L 13.2 14.21 L 22.4 13.77 L 31.59 13.99 L 40.8 13.46 L 50 11.68 L 59.19 11.35 L 68.39 10.68 L 77.6 7.11 L 86.8 7.85 L 96 4 V 20 L 4 20 Z" stroke="none"></path></svg>`;
 
-    elt.innerHTML = '';
-    elt.appendChild(link);
+  elt.innerHTML = '';
+  elt.appendChild(link);
 });
 ```
 
 Custom block entities can also be created (have a look at the separate [Draftail documentation](https://www.draftail.org/docs/blocks)), but these are not detailed here since [StreamField](streamfield_topic) is the go-to way to create block-level rich text in Wagtail.
 
+## Other editor extensions
+
+Draftail has additional APIs for more arbitrary customisations:
```

Comment:
The word "arbitrary" is used so much in this section, I'm beginning to question whether I correctly understand what it means. Could we replace some of the uses with another term?

Reply:
Bespoke?